app/src/main/java/com/grzegorzdec/tagmango/foodie/MenuFragment.kt

```diff
@@ -21,13 +21,13 @@ class MenuFragment : BaseFragment() {
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
-    ): View? =
```

Comment:
what is wrong here? 😃

Reply:
Oh I think those are remainings when this method had more than just one line when I accidentally was changing wrong Fragment :p